### PR TITLE
Show upload claiming file error

### PR DESCRIPTION
**Pull request by gabaldon**
This PR shows an error when no claiming file is uploaded and keep the claiming file available before it is saved in the wallet.

### CODE REVIEW

File: src/components/claiming/UploadFile.vue [hunk 1]
```diff
@@ -21,19 +21,25 @@
         :http-request="handleUpload"
         :on-change="handleUpload"
         :limit="1"
+        :file-list="file"
```
Comment (Tommytrg) [inline src/components/claiming/UploadFile.vue]: Is there a way to change the UI when the file is uploaded?

File: src/components/claiming/UploadFile.vue [hunk 2]
```diff
@@ -21,19 +21,25 @@
...
         :on-exceed="handleExceed"
+        :on-remove="clearClaimingInfo"
       >
         <i class="el-icon-upload"></i>
-        <div class="el-upload__text">Suelta tu archivo aquí o <em>haz clic para cargar</em></div>
+        <div class="el-upload__text">Drag your file here or <em>click to upload</em></div>
         <div slot="tip" class="el-upload__tip">Only application/json files supported</div>
       </el-upload>
+      <p v-if="uploadFileError" class="error">
```
Comment (Tommytrg) [inline src/components/claiming/UploadFile.vue]: If you upload a file once and you try to upload it again it shows an error. The expected behaviour is allow upload the file as many times as you want

File: src/components/claiming/UploadFile.vue (hunk 1 above)
Comment (gabaldon) [reply src/components/claiming/UploadFile.vue]: It already changes. When the file is uploaded a green tick is shown to the user.

File: src/components/claiming/UploadFile.vue (hunk 2 above)
Comment (gabaldon) [reply src/components/claiming/UploadFile.vue]: You need to delete the current file and upload it again. You cannot upload more than one.

File: src/components/claiming/UploadFile.vue (hunk 1 above)
Comment (gabaldon) [reply src/components/claiming/UploadFile.vue]: Discuss in Issue #980

**Pull request closed without merging**